### DISCUSSION TITLE
账号封禁

### DISCUSSION THREAD

**Discussion by stevenli11** [BurnHair/docs#187] [技术支持]
不知道为什么账户被封禁了，我之前反映我改了Github的名字，然后就帮我把额度转移了，里面还有几十刀的额度，但是现在被封禁了：用户9790被封禁1968连坐是啥意思，还有telegram怎么能重新绑定我现在的账号呢。

**Comment by BennyThink** [comment]
你和1986用了一样的IP，1986做了非常恶劣的事情。

所以……

**Reply by stevenli11** [reply]
啊，不应该啊，账号只有我一个人在用，而且我现在在美国😂

**Comment by BennyThink** [comment]
已经启用了，换绑telegram 发邮件

**Comment by stevenli11** [comment]
还有，我想问一下，因为我改了Github的名字，之前的几十刀的额度不知道转到哪去了呢😂

**Comment by BennyThink** [comment]
你之前的号是什么，如果转移了的话应该有的吧

**Reply by stevenli11** [reply]
我忘记了什么时候转的来着，之前的名字是Leven1113

**Reply by BennyThink** [reply]
😂估计是手误转给别人了。你还记得Leven1113有多少额度吗？

**Reply by stevenli11** [reply]
至少应该有30-40的额度来着
